pkg/cloud/baremetal/actuators/machine/actuator.go

```diff
@@ -910,7 +910,10 @@ func (a *Actuator) requestPowerOff(ctx context.Context, baremetalhost *bmh.BareM
 		return &machineapierrors.RequeueAfterError{RequeueAfter: time.Second * 5}
 	}
 
-	baremetalhost.Annotations[requestPowerOffAnnotation] = ""
+	// Issue a hard reboot for immediate remediation purposes
+	remediationRebootAnnotation := bmh.RebootAnnotationArguments{Mode: bmh.RebootModeHard}
+	remediationString, _ := json.Marshal(remediationRebootAnnotation)
```

Comment:
It's best not to ignore errors. It's not likely, but it could be an OOM or something.